### ISSUE TITLE
Removing App.Config from Source Control 

### ISSUE THREAD

**Issue by dlee1964** [Microsoft/slow-cheetah#111]
When I removed my App.Config from source control, SlowCheetah no longer transforms the xxx.exe.config file.  I can still preview the transform and the preview shows the correct information.  I've tried adding back to source control and reinstalling SlowCheetah from nuget, but I cannot get this to work again.  What do I need to do?

**Comment by icnocop** [comment]
Is your project file configured like the [Transforming Files with SlowCheetah](https://github.com/Microsoft/slow-cheetah/blob/master/doc/transforming_files.md) documentation?

```
<None Include="App.config">
    <TransformOnBuild>true</TransformOnBuild>
</None>
<None Include="App.Debug.config">
    <DependentUpon>App.config</DependentUpon>
    <IsTransformFile>True</IsTransformFile>
</None>
<None Include="App.Release.config">
    <DependentUpon>App.config</DependentUpon>
    <IsTransformFile>True</IsTransformFile>
</None>
```

**Comment by dlee1964** [comment]
Yes, the project file is structured as outlined above.  Further information: Transforms work in all but one of my projects.  One thing I am noticing in the project where the transforms are not working is the above structure is not isolated within its own ItemGroup whereas in the working projects they are isolated in its own ItemGroup.  Could this be the issue?

**Comment by dlee1964** [comment]
For the one project in question, the config file is present in the output without transformations.  Again, for all other projects the transforms are as expected.

**Comment by dlee1964** [comment]
Ok, here is additional information.  The project name is NoticesForm (again, one of 5 projects in the solution).  In the obj\release folder of this one project, the file "NoticesForm.csproj-sc.App.config" transforms properly, but the "NoticesForm.exe.config" file does not get transformed at all.  Confusingly, the "NoticesForm.exe.config" in the bin\release folder transforms properly! Each of these files are in the same project (so thus depend on the same app.config files and the same .csproj file if my understanding is correct).  

FWIW, I have changed my source control from TFVC to Git, and along the way, I noticed that some of the app.configs had the "App" part of the name capitalized and others were lowercase.  I have made them all consistent to be lowercase, and have changed the references in the .csproj file to be all lower case as well.  Lastly, I have upgraded to the latest SlowCheetah package (3.1.66).

Is there any way I can troubleshoot why this one file still is not transformed while all the others in the project and entire solution seem to be transforming properly?

**Comment by icnocop** [comment]
Try increasing the verbosity of MSBuild project build output.
